src/components/Home.jsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from "react";
+
+import { getArticles } from "../api";
+import ArticlesList from "./ArticlesList";
+
+const Home = ({ articlesList, setArticlesList }) => {
+  useEffect(() => {
+    getArticles().then((articles) => {
```

Comment:
Needs handling/visual feedback implementation for when requests are in progress with some loading state. Right now there's a lot of blank screen happening whilst waiting for data to load on slower connections. Check the notes for useEffect out again if you need a refresher!